app/templates/components/archive-project-card.hbs

```diff
@@ -15,7 +15,11 @@
       <div class="text-center medium-margin-right medium-margin-left">
         <p class="lead small-margin-bottom">
           <strong>
-            {{this.assignment.project.dcpPublicstatusSimp}}
+            {{#if (eq this.assignment.project.dcpPublicstatus 'Withdrawn/Terminated/Disapproved')}}
+              Withdrawn, Terminated, or Disapproved
+            {{else}}
+              {{this.assignment.project.dcpPublicstatus}}
+            {{/if}}
```

Comment:
At a later point we'll want to come up with a metadata system that maps domain values to presentable ones